frontend/src/scenes/onboarding/sdks/session-replay/js-web.tsx

```diff
@@ -0,0 +1,40 @@
+import { JSSnippet } from 'lib/components/JSSnippet'
+import { LemonDivider } from 'lib/lemon-ui/LemonDivider'
+import { CodeSnippet, Language } from 'lib/components/CodeSnippet'
+import { useValues } from 'kea'
+import { teamLogic } from 'scenes/teamLogic'
+import { JSInstallSnippet, SessionReplayFinalSteps } from '../shared-snippets'
+
+function JSSetupSnippet(): JSX.Element {
+    const { currentTeam } = useValues(teamLogic)
+
+    return (
+        <CodeSnippet language={Language.JavaScript}>
+            {[
+                "import posthog from 'posthog-js'",
+                '',
+                `posthog.init('${currentTeam?.api_token}', { api_host: '${window.location.origin}' })`,
+            ].join('\n')}
+        </CodeSnippet>
+    )
+}
+
+export function JSWebInstructions(): JSX.Element {
+    return (
+        <>
+            <h3>Option 1. Code snippet</h3>
+            <p>
+                Just add this snippet to your website within the <code>&lt;head&gt;</code> tag and we'll automatically
```

Comment:
I agree with your comment in #dev where you suggest restyling this a little better